README.md

```diff
@@ -44,6 +44,16 @@ MIT-Lizenz lizenziert sind.
 Bitte stelle sicher, dass deine Änderungen getestet wurden, bevor du einen Pull-Request sendest.
 
 # For Developers 👩‍💻 👨‍💻
+
+## Prerequisites
+
+The following prerequisites must be installed:
+
+- python 3.9
+- pipenv
+- postgresql
```

Comment:
Maybe also specifying that it is the postgresql client that is needed (in ubuntu the package is called postgrtesql-client), meaning only the client needs to be installed and not the whole contrib package (postgresql-contrib).
And in addition also maybe the minimum version needed --> 9.1, see [here](https://pytest-pgsql.readthedocs.io/en/latest/installation.html).

Reply:
Good point! Done.